model.py

```diff
@@ -1,19 +1,56 @@
+from dao import Dao
+
 class Model:
+    data_access = Dao()
 
-    def __init__(self, name, function):
+    def __init__(self, name, function, _id):
         self.name = name
         self.function = function
+        self._id = _id
+    @classmethod
+    def update(cls, name, function):
```

Comment:
lets pass in a collection and allow the update to work on all `key: value` pairs

Reply:
done